rflx/model/message.py

```diff
@@ -1632,7 +1679,28 @@ def prune_dangling_fields(
                 break
 
             field, inner_message = inner_messages.pop(0)
-            inner_message = inner_message.prefixed(f"{field.name}_")
+            inner_message, message_attribute_locations = replace_message_attributes(
+                inner_message.prefixed(f"{field.name}_")
+            )
+
+            message_next_fields = message.outgoing(field)
+            if len(message_attribute_locations) > 0 and (
+                len(message_next_fields) > 1 or message_next_fields[0].target != FINAL
+            ):
```

Comment:
Would it be a problem if there would be multiple links to `FINAL`?

Reply:
Good catch. Actually this should be no problem as we cannot have size attributes on the links to final and the condition cannot alter the position of `Message'Last`. I changed that in 877fd78d to support multiple links to FINAL.